src/common/LatencyTuner.cpp

```diff
@@ -55,7 +55,7 @@ Result LatencyTuner::tune() {
             if ((xRunCountResult.value() - mPreviousXRuns) > 0) {
                 mPreviousXRuns = xRunCountResult.value();
                 int32_t oldBufferSize = mStream.getBufferSizeInFrames();
-                int32_t requestedBufferSize = oldBufferSize + mStream.getFramesPerBurst();
+                int32_t requestedBufferSize = oldBufferSize + mStream.getFramesPerBurst(); // Aren't we afraid of potential integer overflows here?
```

Comment:
No. The bufferSize is usually under 10K and is limited to the bufferCapacity. It cannot grow unbounded.

Reply:
Is there some constant throttling it under 10K, or what gives us confidence that it cannot be bigger than that?